templates/electron-basic/webpack.config.babel.js

```diff
@@ -0,0 +1,123 @@
+import path from 'path';
+import webpack from 'webpack';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+
+const env = process.env.NODE_ENV || 'production';
+
+let plugins = [
+  new CopyWebpackPlugin([
+    { from: './public' },
+    { from: './electron-entry.js' },
+    { from: './package.json' },
```

Comment:
Lines 10,11 are changed from the original -- to copy the electron entrypoint script and package.json into dist